app/blueprints/round/routes.py

```diff
@@ -197,12 +198,16 @@ def clone_round(round_id):
     return redirect(url_for(ROUND_DETAILS, round_id=round_id))
 
 
-@round_bp.route("/<round_id>")
+@round_bp.route("/<round_id>", methods=["GET", "DELETE"])
 def round_details(round_id):
     fund_round = get_round_by_id(round_id)
     form = RoundForm(data={"fund_id": fund_round.fund_id})
-    cloned_form = CloneRoundForm(data={"fund_id": fund_round.fund_id})
     fund_form = FundForm()
+    if request.method == "DELETE":
+        delete_selected_round(round_id)
+        return redirect(url_for("round_bp.view_all_rounds"))
```

Comment:
I think we should just remove the delete handling in both of these HTTP views altogether until we have worked out the UI for it.

It's still useful to have done the work writing the helpers functions and we (devs) can use these if needed. But let's not hook up anything on the site until we have decided what we're doing.